contact/views.py

```diff
@@ -0,0 +1,19 @@
+from django.http import Http404
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from contact.models import Information
+from contact.serializers import InformationSerializer
+
+
+class InformationDetail(APIView):
+    def get_object(self, pk):
+        try:
+            return Information.objects.get(pk=pk)
+        except Information.DoesNotExist:
+            raise Http404
+
+    def get(self, request, pk):
+        info = self.get_object(pk)
+        serializer = InformationSerializer(info)
+        return Response(serializer.data)
```

Comment:
Thanks for the `get_object_or_404`, I totally forgot about it. 

Using `ReadOnlyModelViewSet` could be an option but as I don't want that the user can list all the info stored in the DB, an APIView just with the get method defined fit perfectly.

Reply:
Okay, finally I decided to replace `APIView` for  `GenericViewSet, RetrieveModelMixin`. It does the same but with less code and using features of ViewSet. As Guillermo explains to me, this allows me to extend the behavior in case the API needs changes in the future.